rewrite-java/src/main/java/org/openrewrite/java/format/TabsAndIndentsProcessor.java

```diff
@@ -36,56 +36,73 @@ public TabsAndIndentsProcessor(TabsAndIndentsStyle style) {
     @Override
     public Statement visitStatement(Statement statement, P p) {
         Statement s = statement;
-        Cursor parentCursor = getCursor().getParentOrThrow();
-        J parent = parentCursor.getTree();
-
-        if (!(s instanceof J.Block)) {
-            parent = parentCursor.getTree();
-            Cursor cursor = parentCursor;
-
-            // find the first cursor element that is indented further to the left
-            for (;
-                 parent instanceof J.Block || parent instanceof J.Label || parent instanceof J.Try.Catch ||
-                         parent instanceof J.If && cursor.getParentOrThrow().getTree() instanceof J.If.Else ||
-                         parent instanceof J.If.Else;
-                 parent = cursor.getTree()) {
-                cursor = cursor.getParentOrThrow();
+
+        if (!useContinuationIndent(s)) {
```

Comment:
I think the parent of the Statement might actually be a J.Block (I think you want to find the block's parent?)

Reply:
I have been adding a watch to the debugger: getCursor()

That is pretty helpful